enhancements/update/automatic-updates.md

```diff
@@ -0,0 +1,166 @@
+---
+title: neat-enhancement-idea
+authors:
+  - "@wking"
+reviewers:
+  - "@abhinavdahiya"
+  - "@crawford"
+  - "@smarterclayton"
+approvers:
+  - TBD
+creation-date: 2019-11-19
+last-updated: 2019-11-19
+status: implementable
+---
+
+# Automatic Updates
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [x] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Operator-managed updates are [one of the key benefits of OpenShift 4][architecture].
+Cluster update suggestions are distributed via Cincinnati][cincinnati-openshift], so clusters only receive update recommendations that are appropriate for their cluster.
+While manually approving updates at a per-cluster level is possible, it should not be required.
+This enhancement adds a new `automaticUpdates` property to [the ClusterVersion `spec`][api-spec] so cluster adminstrators can opt in to or out of automatic updates.
+
+## Motivation
+
+Manually approving updates is tedious, doesn't scale well for administrators managing many clusters, and delays the application of potential security fixes.
+Administrators may wish to enable automatic updates for any of those or other reasons.
+Currently, administrators must implement custom polling logic to check for and apply any available updates.
+This enhancement would provide those administrators with a convenient property instead.
+
+### Goals
+
+* Make it easy to opt in to and out of automatic updates.
+
+### Non-Goals
+
+* Support calendar gating or other logic about when updates may be attempted.
+* Choose whether the installer defaults to enabling automatic updates or not for new clusters.
+
+## Proposal
+
+As proposed in [this API pull request][api-pull-request], to add a new property:
+
+```go
+// automaticUpdates enables automatic updates.
+// +optional
+AutomaticUpdates bool `json:"automaticUpdates,omitempty"`
+```
+
+to [`ClusterVersionSpec`][api-spec] to record the administrator's preference.
+
+### User Stories
+
+#### Canary Clusters
+
+Alice has a canary clusters she wants to update as soon as an updated release is available in her configured channel.
```

Comment:
I'd rather not use the term "canary" here, as it has some implications on how release-engineering fit into this.
My suggestion would be to focus on the "leading edge" part, or similar.

Reply:
> I'd rather not use the term "canary" here...

Avoided with dae7d51 -> 9b61a96.